package.json

```diff
@@ -48,7 +48,7 @@
     "start:worker": "gulp worker:dev",
     "start": "concurrently --raw --kill-others --kill-others-on-fail 'npm:lint:watch' 'npm:start:develop' 'npm:start:worker' 'npm:dashboard:mail'",
     "test:client:watch": "gulp angular-templatecache && npm run check-environment && NODE_ENV=test karma start",
```

Comment:
Should you add `jest --watch` here in `test:client:watch`? It should run in parallel with Karma so likely would need to use `concurrently`.

Reply:
Possibly, I left it out intentionally, but not for a particularly good reason, just to avoid having to think about it :) ... I'll try it out (it would be `jest --watchAll` though).

Reply:
Following up in separate PR is fine, totally not a blocker for merging. 👍 

Oh right, I thought `test:client:watch` re-runs only tests that changed but looks like it indeed re-runs everything. :-)

Reply:
The way jest outputs the results isn't very compatible with running it concurrently with the karma tests as jest clears the console, assuming it's the only thing in the terminal. I looked for some option that made it just output linearly, but no joy.

Reply:
Ahh, bummer! Thanks for checking.

Suppose folks can just run `npx jest --watch` directly in the root when working on those tests? 😁

Reply:
> run `npx jest --watch` directly
Yes. Who wants to run all the tests when doing BDD anyways? :slightly_smiling_face:

I would enjoy having a script in package.json, then. Like `"test:react": "jest"`. Maybe also script for `test:react:watch`, even though it can be achieved with "npm run test:react -- --watchAll".

Not a blocker. :slightly_smiling_face:

Reply:
I'm slightly hesitant to add extra scripts, as eventually it would use the existing ones (`npm run test:client` and `npm run test:client:watch`) the jest/react ones would only be temporary until the test runners are unified.

Also, `npm run test:client` already runs the jest tests, it's just the watching that doesn't.

So, unless someone would _really_ like extra scripts, I would leave it as it is for now, and hopefully get the test runners unified in not _forever_...

Reply:
> hopefully get the test runners unified in not forever...

Haha, deal!